src/boot/efi/boot.c

```diff
@@ -746,6 +746,9 @@ static bool menu_run(
                                 lines[i] = xnew(char16_t, line_width + 1);
                                 padding = (line_width - MIN(strlen16(config->entries[i]->title_show), line_width)) / 2;
 
+                                /* Make sure there is space for => */
+                                padding = MAX((size_t)2,padding);
```

Comment:
to follow coding style, please add space between `(size_t)` and `2`. Also please add space after `,`